.travis.yml

```diff
@@ -1,10 +1,15 @@
-sudo: required
-dist: trusty
+dist: xenial
 services:
   - docker
 language: rust
 
+git:
+  # FIXME: Issue #1783
+  #depth: 1
```

Comment:
Is there a significant speed benefit in shallow cloning here?

Reply:
Neglectable :) I just wonder if someone download zipped of rustup
and run `cargo test`, then it would be failed, which will confuse
them.

Reply:
The bug report is fair, that's a problem for me to fix, but for this, I'm not sure we'd want depth:1 anyway since we *want* git history information.